### ISSUE TITLE
Page missing headings

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#682]
[Page missing headings - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402505/Page.missing.headings.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)

**Comment by AinaEngen** [comment]
Jim jobber med å fikse overskriftene

**Comment by AinaEngen** [comment]
Gjenstår 5 steder hvor dette nå oppstår - dette er maskinkode og har derfor ingen overskrifter. Foreslår at vi skriver en kommentar i tilgjengelighetserklæringen om dette
